packages/snap-preact-components/src/components/Molecules/Radio/Radio.tsx

```diff
@@ -0,0 +1,188 @@
+/** @jsx jsx */
+import { h } from 'preact';
+import { useState, StateUpdater } from 'preact/hooks';
+
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+import { observer } from 'mobx-react-lite';
+
+import { ComponentProps, StylingCSS } from '../../../types';
+import { defined } from '../../../utilities';
+import { Theme, useTheme, CacheProvider } from '../../../providers';
+import { Icon, IconProps } from '../../Atoms/Icon';
+import { useA11y } from '../../../hooks/useA11y';
+import { mergeProps } from '../../../utilities';
+
+const CSS = {
+	radio: ({ size, color, theme }: Partial<RadioProps>) =>
+		css({
+			display: 'inline-flex',
+			alignItems: 'center',
+			justifyContent: 'center',
+			height: size,
+			width: size,
+			cursor: 'pointer',
+
+			'&.ss__radio--disabled': {
+				opacity: 0.5,
+			},
+			'& .ss__radio__empty': {
+				display: 'inline-block',
+				width: `calc(${size} - 30%)`,
+				height: `calc(${size} - 30%)`,
+			},
+
+			'& .ss__radio__icon--active circle:first-of-type': {
```

Comment:
These color styles should be unnecessary when making the suggested change to the icon path in this review.

Reply:
seemed to be needed still.